api/app/signals/apps/api/v1/urls.py

```diff
@@ -29,6 +30,18 @@
     basename='public-signals'
 )
 
+signal_router_v1.register(
+    r'public/feedback/standard_answers',
```

Comment:
Minor thing but why use an underscore instead of a hyphen?

Reply:
will leave